src/monitorService.ts

```diff
@@ -74,7 +78,7 @@ export class TxMonitorService {
 
   private async resendTransaction(transaction: SavedTransactionResponse): Promise<void> {
     const newGasPrice = await recalculateGasPrice(
-      transaction.gasPrice,
+      transaction.gasPrice ?? BigNumber.from(0),
```

Comment:
?

Reply:
Ethers changed type aparently gasPrice can be undefined now